easybuild/easyconfigs/p/PyTorch3D/PyTorch3D-0.4.0-fosscuda-2020b-PyTorch-1.8.1.eb

```diff
@@ -0,0 +1,51 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'PyTorch3D'
+version = '0.4.0'
+local_pytorch_version = '1.8.1'
+versionsuffix = '-PyTorch-%s' % local_pytorch_version
+
+homepage = "https://pytorch3d.org/"
+description = """PyTorch3D is FAIR's library of reusable components for deep learning with 3D data."""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('tqdm', '4.56.2'),
+    ('PyTorch', local_pytorch_version),
+    ('imageio', '2.9.0'),
+    ('IPython', '7.18.1'),
+    ('torchvision', '0.9.1', versionsuffix),
```

Comment:
@branfosj This should be lowered to 0.8.2 to stick to the single-dependency-version per easyconfigs generation rule?

Reply:
Yes, which will also put it against Pytorch 1.7.1. Oddly, that is what we have installed locally, but I must have bumped the version when pushing upstream.

Reply:
fixed in bd3aa47